Nodes/SerialIO.c

```diff
@@ -425,14 +434,19 @@ void serreqtkn(void) {
 
 void putstring(char *pekare,int size, long flags) {
 	int err;
-	char serpekare[200];
+	char serpekare[400];
+	int bytes;
 
-	strncpy(serpekare,pekare,199);
-	serpekare[199] = 0;
-	if(!(flags & PS_NOCONV)) ConvChrsFromAmiga(serpekare,0,Servermem->inne[nodnr].chrset);
+	if((flags & PS_NOCONV)) {
```

Comment:
There is no call to putstring() that actually use the PS_NOCONV flag. And I honestly can't remember what that flag would be good for in the first place. Please remove the checking of the flag and just assume that it's always 0.